src/createAction.js

```diff
@@ -29,7 +29,7 @@ export default function createAction(type, payloadCreator, metaCreator) {
     return action;
   };
 
-  actionHandler.toString = () => type;
+  actionHandler.toString = () => type.toString();
```

Comment:
As we are currently supporting them in `handleActions`, I would keep the support here as well.